source/_integrations/zha.markdown

```diff
@@ -414,9 +414,23 @@ In this theoretical example, a CC2652-based Zigbee coordinator has three CC2530
 
 In practice, you will likely need to add a lot more Zigbee router devices than in this example in order to extend the coverage of the network to reach that many devices.
 
-## Binding and unbinding
+## Zigbee Groups, Bindings and Unbindings
 
-ZHA support for binding and unbinding. Binding is an action in Zigbee which defines relations between two Zigbee devices, specific endpoints, and cluster id. It provides a mechanism for attaching an endpoint on one Zigbee node to one or more endpoints on another Zigbee node or Zigbee group (a group of Zigbee devices).
+ZHA integration support for native Zigbee Groups, as well as Zigbee Bindings and Unbindings. These features can be used separately or combined.
+
+### Zigbee Group
+
+A "Zigbee Group" enables native on-device grouping of multiple Zigbee lights, switches, and fans that enable controlling all compatible entities for those devices in those groups with only one command/entity.
+
+Importantly note that using a native "Zigbee Group" inside the ZHA integration instead of Home Assistant's [Group](/integrations/group/) integration should make for much faster response times and better synchronization, especially when grouping lights.
```

Comment:
I think the "importantly" sound a bit off. Maybe change the phrasing to something like "Note that..." and put the whole paragraph into a callout block (if it's really that important).

```suggestion
<div class='note'>
Note that using a native "Zigbee Group" inside the ZHA integration instead of Home Assistant's [Group](/integrations/group/) integration should make for much faster response times and better synchronization, especially when grouping lights.
</div>
```